### PR TITLE
bcda-3170 Feature: Add Attribution Status endpoint(s)

### PR DESCRIPTION
**Pull request by SMLuthi**
### Fixes [BCDA-3170](https://jira.cms.gov/browse/BCDA-3170)

Currently ACOs have no way to identify the last time a cclf ingestion has taken place. To keep them from spamming jobs and resulting in the same data we want to open an end point that allows them to see the last time we ingested their cclf file (and runout file). This way they can time their export jobs better and avoid rerunning jobs resulting in the same data.

### Proposed Changes

Create new endpoints (for v1 and v2) that returns timestamps for the most recently ingested cclf 8 default and runout files.

### Change Details

- Create `<host>/api/v1/attribution_status` endpoint
- Create `<host>/api/v2/attribution_status` endpoint
- Add swagger docs
- Add Postman tests

### Security Implications

- [ ] new software dependencies

- [ ] security controls or supporting software altered

- [ ] new data stored or transmitted

- [x] security checklist is completed for this change
[Security Checklist](https://confluence.cms.gov/display/BCDA/21+-+07+-+28%3A+Attribution+Status+Endpoint+Security+Checklist)

- [ ] requires more information or team discussion to evaluate security implications

- [x] no PHI/PII is affected by this change

### Acceptance Validation

Unittests pass. 
Postman tests pass. 
Validate that swagger documents look correct.

### Feedback Requested

The usual, whatever you think is worth addressing.

### CODE REVIEW

**Comment by msnook** [comment]
@SMLuthi Can you link to the security checklist in the `Security Implications` section?

File: bcda/api/v2/api_test.go [hunk 1]
```diff
@@ -470,6 +470,28 @@ func (s *APITestSuite) TestResourceTypes() {
 	}
 }
 
+func (s *APITestSuite) TestGetAttributionStatus() {
+	req := httptest.NewRequest("GET", "/api/v1/attribution_status", nil)
```
Comment (CosmicSyntax) [inline bcda/api/v2/api_test.go:474]: Should this be v2?

**Review by CosmicSyntax — APPROVED** [review]
🎉

File: Dockerfiles/Dockerfile.documentation [hunk 2]
```diff
@@ -26,6 +26,6 @@ CMD mkdir -p swaggerui/v2 && \
     mkdir -p swaggerui/v1 && \
     cp -r /tmp/swagger-ui-${swaggerVersion}/dist/* swaggerui/v1/ && \
     cp -r /tmp/swagger-ui-${swaggerVersion}/dist/* swaggerui/v2/ && \
-    swagger generate spec -o swaggerui/v1/swagger.json -m --include-tag metadata --include-tag bulkData --include-tag auth --include-tag job && \
-    swagger generate spec -o swaggerui/v2/swagger.json -m --include-tag metadataV2 --include-tag bulkDataV2 --include-tag auth --include-tag jobV2 && \
+    swagger generate spec -o swaggerui/v1/swagger.json -m --include-tag metadata --include-tag bulkData --include-tag auth --include-tag job --include-tag attributionStatus && \
```
Comment (katyasoup) [inline Dockerfiles/Dockerfile.documentation:29]: 🎉

**Review by katyasoup — APPROVED** [review]
LGTM!

**Review by jphechter — APPROVED** [review]
LGTM! 👍

**Pull request merged**